brainscore/benchmarks/zhu2019.py

```diff
@@ -0,0 +1,170 @@
+import numpy as np
+from numpy import ndarray
+from numpy.random import RandomState
+
+import brainscore
+from brainio.assemblies import DataAssembly
+from brainscore.benchmarks import BenchmarkBase
+from brainscore.benchmarks.screen import place_on_screen
+
+from brainscore.metrics import Score
+from brainscore.metrics.accuracy import Accuracy
+from brainscore.model_interface import BrainModel
+from brainscore.utils import LazyLoad
+from brainscore.metrics.error_consistency import ErrorConsistency
+
+BIBTEX = """@article{zhu2019robustness,
+            title={Robustness of object recognition under extreme occlusion in humans and computational models},
+            author={Zhu, Hongru and Tang, Peng and Park, Jeongho and Park, Soojin and Yuille, Alan},
+            journal={arXiv preprint arXiv:1905.04598},
+            year={2019}
+        }"""
+
+DATASETS = ['extreme_occlusion']
+
+# create functions so that users can import individual benchmarks as e.g. Zhu2019RDM
+for dataset in DATASETS:
+    # behavioral benchmark
+    identifier = f"Zhu2019{dataset.replace('-', '')}Accuracy"
+    globals()[identifier] = lambda dataset=dataset: _Zhu2019Accuracy(dataset)
+
+    # engineering benchmark
+    identifier = f"Zhu2019{dataset.replace('-', '')}AccuracyEngineering"
+    globals()[identifier] = lambda dataset=dataset: _Zhu2019Accuracy_Engineering(dataset)
+
+
+class _Zhu2019Accuracy(BenchmarkBase):
```

Comment:
same here, if it's only one dataset, you can remove the `init` parameter and just make this into `class Zhu2019Accuracy`